src/atom_finder/classifier/logic-as-control-flow.clj

```diff
@@ -5,19 +5,19 @@
   "can this AST node change program state?"
   [node]
   (let [b-ops #{ ; binary operators with side-effects
-          IASTBinaryExpression/op_assign          IASTBinaryExpression/op_binaryAndAssign
-          IASTBinaryExpression/op_binaryXorAssign IASTBinaryExpression/op_divideAssign
-          IASTBinaryExpression/op_minusAssign     IASTBinaryExpression/op_moduloAssign
-          IASTBinaryExpression/op_multiplyAssign  IASTBinaryExpression/op_plusAssign
-          IASTBinaryExpression/op_shiftLeftAssign IASTBinaryExpression/op_shiftRightAssign}
+                IASTBinaryExpression/op_assign          IASTBinaryExpression/op_binaryAndAssign
+                IASTBinaryExpression/op_binaryXorAssign IASTBinaryExpression/op_divideAssign
+                IASTBinaryExpression/op_minusAssign     IASTBinaryExpression/op_moduloAssign
+                IASTBinaryExpression/op_multiplyAssign  IASTBinaryExpression/op_plusAssign
+                IASTBinaryExpression/op_shiftLeftAssign IASTBinaryExpression/op_shiftRightAssign}
         u-ops #{ ; unary operators with side-effects
-          IASTUnaryExpression/op_postFixDecr      IASTUnaryExpression/op_postFixIncr
-          IASTUnaryExpression/op_prefixDecr       IASTUnaryExpression/op_prefixIncr}]
-  (cond
-    (leaf? node) false
-    (instance? IASTBinaryExpression node) (contains? b-ops (.getOperator node))
-    (instance? IASTUnaryExpression node) (contains? u-ops (.getOperator node))
-    :else (= (typename node) "FunctionCallExpression"))))
+                IASTUnaryExpression/op_postFixDecr      IASTUnaryExpression/op_postFixIncr
+                IASTUnaryExpression/op_prefixDecr       IASTUnaryExpression/op_prefixIncr}]
+   (cond
+     (leaf? node) false
+     (instance? IASTBinaryExpression node) (contains? b-ops (.getOperator node))
+     (instance? IASTUnaryExpression node) (contains? u-ops (.getOperator node))
+     :else (= (typename node) "FunctionCallExpression"))))
```

Comment:
In general you should avoid committing white-space only changes. It screws up the history.